versioned_docs/version-21.10/monitoring/monitoring-servers/add-a-remote-server-to-configuration.md

```diff
@@ -63,6 +63,19 @@ Remote Server on the Centreon platform.
 
   ![image](../../assets/monitoring/monitoring-servers/remote-list-zmq.png)
 
+6. If you have customized the names of the databases at step 6 of the [web installation wizard](../../installation/web-and-post-installation.md), carry out the following steps:
+
+  1. Go to **Configuration > Pollers > Broker configuration**.
+  2. Select the broker service for the remote server you want.
+  3. On the **Output** tab, in section **Output 1 - Unified SQL**, update the name of the database in the **DB name** field (the default name is **centreon_storage**), then click **Save**.
+  4. [Export the configuration](deploying-a-configuration.md) of the remote server.
+  5. Restart cbd:
+
+     ```shell
+     systemctl restart cbd
+     ```
+  If needed, check **/var/log/centreon-broker/remote1.log** for issues/errors.
```

Comment:
the name remote1.log should be replaced by the name for the remote used in the screenshots